doc/PREFERENCE.md

```diff
@@ -0,0 +1,48 @@
+## Mark Text Preference
+
+#### General
+
+| Key                 | Type    | Default Value | Description                                                                                                                                                |
+| ------------------- | ------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| autoSave            | Boolean | ture          | Automatically save the content being edited. option value: true, false                                                                                     |
+| autoSaveDelay       | Number  | 3000          | How long do you want to save your document(ms)?3000 ~10000                                                                                                 |
```

Comment:
How about: The delay in milliseconds after a changed file is saved automatically.